packages/gamut/src/Markdown/__tests__/Markdown.test.tsx

```diff
@@ -41,21 +42,21 @@ const checkboxMarkdown = `
 - [ ] third checkbox
 `;
 
-const renderWrapper = setupEnzyme(Markdown);
+const renderView = setupRtl(Markdown);
 
 describe('<Markdown />', () => {
   it('renders standard Markdown', () => {
-    const { wrapper } = renderWrapper({ text: basicMarkdown });
-    expect(wrapper.find('h1').length).toEqual(1);
-    expect(wrapper.find('h3').length).toEqual(1);
-    expect(wrapper.find('code').length).toEqual(1);
+    renderView({ text: basicMarkdown });
+    expect(document.querySelectorAll('h1').length).toEqual(1);
```

Comment:
i think you could do this instead if you wanted to use more of the built in rtl matchers
```suggestion
    view.getByRole('heading', { level: 1 })
```